core/header.cpp

```diff
@@ -1,3 +1,4 @@
+<<<<<<< HEAD:core/header.cpp
```

Comment:
:poop: I'll fix this, but check whether the rest is ok of course. :slightly_smiling_face: